src/base/bittorrent/session.cpp

```diff
@@ -1124,6 +1124,7 @@ void Session::configure(libtorrent::settings_pack &settingsPack)
             }
         }
 
+        settingsPack.set_str(libt::settings_pack::outgoing_interfaces, networkInterface().toStdString());
```

Comment:
>That code sets multiple listening IPs in a loop, while here we want to pass list of interfaces for outgoing connections

From what I see, that loop selects only 1 interface to listen to.
And I thought the listened on interface should also be the one outgoing, hence the speculation.

If it works now then I'm fine with it too.